### PR TITLE
Update origami-studio to 131604203

### PR DESCRIPTION
**Pull request by dragonandy**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by dragonandy** [comment]
Add 'depends_on' statement

File: Casks/origami-studio.rb [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+  version '131604203'
+  sha256 'ba26f6b289adb6d883074c0e42e385b867dff42df4b356a15a2f431e7cac4967'
 
   # fb.me/getorigamistudio was verified as official when first introduced to the cask
   url 'https://fb.me/getorigamistudio'
   appcast 'https://www.facebook.com/mobile_builds/appcast.xml?app_id=892075810923571'
   name 'Origami Studio'
   homepage 'https://origami.design/'
 
+  depends_on macos: '>= 10.12'
```
Comment (vitorgalvao) [inline Casks/origami-studio.rb]: Never a number. Always a name.

Comment (ran-dall) [reply Casks/origami-studio.rb]: @vitorgalvao Noted. 

Just a note, IMO it's not very clear in the docs (https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/depends_on.md), it even has an example `depends_on macos: '>= 10.9'`...or am I missing something? As always, your help is much appreciated.

Comment (vitorgalvao) [reply Casks/origami-studio.rb]: From the document:

> The symbol form is preferred for readability.

Comment (ran-dall) [reply Casks/origami-studio.rb]: 🙏 That would be what I was missing! Thanks!

**Pull request merged**